lib/AdminPages/PageRenderer.php

```diff
@@ -70,6 +76,9 @@ class_exists(Debugger::class)
       ) {
         DIPanel::init();
       }
+      if (is_admin() && $this->subscribersCountCacheRecalculation->shouldBeScheduled()) {
+        $this->subscribersCountCacheRecalculation->schedule();
```

Comment:
I am not sure I understand this change. Why do we need to schedule this calculation in the renderer?

Reply:
The reason is that we don't need to recalculate cache when an admin doesn't need it. I liked that idea from Rosta's [comment](https://wp.me/pcNwfB-3B#comment-189). Maybe my solution is wrong.

Reply:
Right, I didn't remember that comment. I was surprised that it is in the renderer, that is all.